### DISCUSSION TITLE
cityselect/jinan

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#190] [Announcements]
# cityselect/jinan

Part1 工作机会 经过调查发现，济南的互联网环境是挺差的，大的公司几乎不会在济南成规模的招程序员。 如果你是一个应届毕业生，拿到济南阿里、华为、海康、铁路局等 offer 的话，想要离家近生活比较舒服那么值得去。其余的公司我不建议校招直接去，想去可以先去外面的大公司干几年，社招再回来。除非是没有更好的 offer。 另外，如果你想一直待在济南，不如去个有编制的地方，虽然我对技术有执念，但是也不得不承认在山东还是去个有编制的地方香。 阿里巴巴

https://tobebetterjavaer.com/cityselect/jinan.html

**Comment by zonekoko** [comment]
二哥，这个系列还更新吗，能不能更新一下无锡的互联网公司
